frontend/src/metabase/nav/components/AppBar/AppBarLogo.styled.tsx

```diff
@@ -18,11 +26,31 @@ export const LogoLink = styled(Link)`
   &:hover {
     background-color: ${color("bg-light")};
   }
+
+  ${props =>
+    !props.isSmallAppBar &&
+    css`
+      ${AppBarLeftContainer}:hover & {
```

Comment:
Why isn't that handled in `AppBarLarge` itself?

Reply:
re: https://github.com/metabase/metabase/pull/27391#discussion_r1066873257

I think both declarations should stay close together in the same file. We should either move them both or nothing.